src/NetAppFiles/NetAppFiles/Volume/NewNetAppFilesVolume.cs

```diff
@@ -97,12 +98,24 @@ public class NewAzureRmNetAppFilesVolume : AzureNetAppFilesCmdletBase
         public string CreationToken { get; set; }
 
         [Parameter(
+            ParameterSetName = FieldsParameterSet,
+            Mandatory = true,
+            HelpMessage = "The service level of the ANF volume")]
+        [Parameter(
+            ParameterSetName = ParentObjectParameterSet,
+            ValueFromPipelineByPropertyName = true,
```

Comment:
don't use ValueFromPipelineByPropertyName, except for the ResourceId parameter set